python/hidet/graph/operator.py

```diff
@@ -35,7 +35,7 @@ def get_operator_name(op, given_name: Optional[str] = None):
 class Operator:
     """An operator that takes tensor as input and output."""
 
-    def __init__(self, inputs: List[Tensor], attributes: Dict[str, Any], task: Optional[Task]):
+    def __init__(self, inputs: List[Tensor], attributes: Optional[Dict[str, Any]] = None, task: Optional[Task] = None):
```

Comment:
OK that sounds good, I added this change because the code below checks if it is None and assigns it to an empty dict if that's the case. This gave me the wrong impression that you meant to have this optional but forgot to do so

Reply:
I see. In the very beggining, I set the default value of attributes=None, and some operators that should pass the attributes did not pass them. So I make this parameter mandatory later, but forget to change the body of __init__. Feel free to delete the check of attributes==None.